### DISCUSSION TITLE
why appeared multiple "navbar__link--active"

### DISCUSSION THREAD

**Discussion by Chasen-Zhang** [facebook/docusaurus#5810] [Q&A] [answered]
## 💥 Proposal

![image](https://user-images.githubusercontent.com/15354455/139204586-a7e7d4d4-5e2a-4853-a6f0-b782ca49971e.png)
![image](https://user-images.githubusercontent.com/15354455/139204809-5a446736-3c8c-4f9c-aef2-5c54a5b390c2.png)

if i change the {to:'/'}   to {to: '/xxxxxx'} ,   it is ok, but i hope  my home page is '/', what should i do?

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

yes

**Comment by Josh-Cena** [comment, marked as answer]
It's impossible for us to help you without further information about setup, Docusaurus version, etc. Also, GH issues is not for questions. If you feel like it's a bug, please use the bug template; otherwise, let's move to discussions.

**Reply by Josh-Cena** [reply]
The duplicate class issue should be fixed by https://github.com/facebook/docusaurus/pull/5264 already, so it's important that you provide your version.

**Reply by Chasen-Zhang** [reply]
docusaurus  v2
"@docusaurus/core": "^2.0.0-beta.8",
"@docusaurus/preset-classic": "^2.0.0-beta.8",

**Reply by Josh-Cena** [reply]
`{to: '/'}` should redirect to the home page (the page at the base URL) already. How is it not working?

**Reply by Chasen-Zhang** [reply]
![image](https://user-images.githubusercontent.com/15354455/139217818-c23f7b95-39c9-4121-86ee-538e1a295b19.png)

two link--active

**Reply by Josh-Cena** [reply]
Ah, I see. You would need to make the link active on _exactly_ the route path. Try this:
```js
{
  to: '/',
  position: 'left',
  label: '...',
  activeBaseRegex: '^/$',
}
```

**Reply by Chasen-Zhang** [reply]
To take effect!  it is Ok!  thank you!  but I don't know why

**Reply by Josh-Cena** [reply]
https://docusaurus.io/docs/api/themes/configuration#navbar-link

**Reply by slorber** [reply]
Not sure but it might be possible to use `exact: true` (ReactRouter Link prop).
https://reactrouter.com/web/api/NavLink/exact-bool

If this doesn't work, we should probably allow this

**Reply by Fannon** [reply]
Having an `excact: true` options sounds like a good option! This is also what I would have expected from a navbar, that it doesn't behave like a "breadcrumb" in that it marks you the complete path. This would also be more convenient than supplying many base regexes.

How could such an option be passed?

**Reply by yolandaccc** [reply]
> Ah, I see. You would need to make the link active on _exactly_ the route path. Try this:
> 
> ```js
> {
>   to: '/',
>   position: 'left',
>   label: '...',
>   activeBaseRegex: '^/$',
> }
> ```

<img width="243" alt="image" src="https://github.com/user-attachments/assets/1fad5fbc-0bae-4210-8f06-9c301baee8b5">
it looks ok, but when I refresh the website  the issues will come back. I find that when i refresh the website the second link add '/' in website.but i don't know how to write the activeBaseRegex.
<img width="137" alt="image" src="https://github.com/user-attachments/assets/e912bd0b-75c6-4220-bcb5-91bd50e4ce2d">
<img width="148" alt="image" src="https://github.com/user-attachments/assets/adbf79fe-a7b4-4f73-96ba-64a1aa122982">


<img width="206" alt="image" src="https://github.com/user-attachments/assets/b5d97f84-aede-495c-82a0-71d7c4914737">
